t/rt93107.t

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/perl
+
+# Regression test for https://rt.cpan.org/Ticket/Display.html?id=93107
+
+use strict;
+use warnings;
+
+use Test::More;
+
+use IO::Poll;
+
+use IO::Handle;
+
+package NewIOHandle {
```

Comment:
`package NAME BLOCK` is not supported until 5.14

Reply:
Thanks. Fixed!